server/routes.go

```diff
@@ -13,6 +12,7 @@ import (
 	"text/template"
 	"time"
 
+	"dario.cat/mergo"
```

Comment:
This is pretty cool library. I didn't know it existed.

I was trying initially to do with `map[string]interface{}` (or `map[string]any`) but then gave up when it came to the reflection part. it's cool this just works.